src/server/GraphQLServer.ts

```diff
@@ -339,68 +378,85 @@ export class GraphQLServer {
             // Collect error metrics for execution result
             if (executionResult.errors && executionResult.errors.length > 0) {
                 for (const error of executionResult.errors) {
+                    this.logger.error('While processing the request ' +
+                        'the following error occurred: ',
+                    error,
+                    this.determineGraphQLOrFetchError(error),
+                    request)
                     this.increaseFetchOrGraphQLErrorMetric(error, request)
                 }
             }
 
             // Return execution result
-            this.logDebugIfEnabled(`Create response from data ${JSON.stringify(executionResult)}`)
-            return this.sendResponse(response, executionResult)
-        } catch (e: unknown) {
-            this.logDebugIfEnabled(`A GraphQL execution error occurred: ${JSON.stringify(e)}`)
-            this.increaseFetchOrGraphQLErrorMetric(e, request)
-            return this.sendGraphQLExecutionErrorResponse(request, response, e as GraphQLError)
+            this.logDebugIfEnabled(`Create response from data ${JSON.stringify(executionResult)}`,
+                request)
+            return this.sendResponse(response, executionResult, 200, {}, request)
+        } catch (error: unknown) {
+            this.logger.error('While processing the request ' +
+                'a GraphQL execution error occurred',
+            error as GraphQLError,
```

Comment:
See above

Reply:
In the Typescript version we are using this seems not to be possible. I'm not sure if this might get fixed in future Typescript versions but for now it is only possible to use any or unknown in catch clause.
See: "TS1196: Catch clause variable type annotation must be 'any' or 'unknown' if specified."